src/template.njk

```diff
@@ -19,9 +19,9 @@
     {% endblock %}
 
     {% block head %}{% endblock %}
-
     {# The default og:image is added below head so that scrapers see any custom metatags first, and this is just a fallback #}
-    <meta property="og:image" content="{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
+    {# image path needs to absolute e.g. http://wwww.domain.com/.../govuk-opengraph-image.png #}
```

Comment:
Path refers to just the part after the domain - i.e. just the `/.../govuk-opengraph-image.png` bit. How about `assetUrl`?

Reply:
sure. updated